python/src/DistFunc_doc.i.in

```diff
@@ -210,3 +294,85 @@ Examples
 --------
 >>> import openturns as ot
 >>> pval = ot.DistFunc.pPearsonCorrelation(100, 0.3, True)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::logpPoisson
+"Logarithm of the probability function of a Poisson distribution.
+
+Parameters
+----------
+lambda: float, :math:`\lambda\geq 0`
+    The intensity of the Poisson distribution
+k : int
+    The number of success.
+
+Returns
+-------
+logp : float
+    The natural logarithm of the probability to get :math:`k` successes.
+
+Notes
+-----
+This method implements Loader's algorithm, the *fast* and *accurate* method described in [loader2000]_, with the further improvements mentioned in [dimitriadis2016]_.
+Examples
+--------
+>>> import openturns as ot
+>>> p = ot.DistFunc.logpPoisson(5.0, 2)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::DistFunc::dPoisson
+"Probability function of a Poisson distribution.
+
+Parameters
+----------
+lambda: float, :math:`\lambda\geq 0`
+    The intensity of the Poisson distribution
+k : int
+    The number of success.
+
+Returns
+-------
+logp : float
+    The natural logarithm of the probability to get :math:`k` successes.
+
+Notes
+-----
+This method implements Loader's algorithm, the *fast* and *accurate* method described in [loader2000]_, with the further improvements mentioned in [dimitriadis2016]_.
+
+Examples
+--------
+>>> import openturns as ot
+>>> p = ot.DistFunc.dPoisson(5.0, 2)"
+
+%feature("docstring") OT::DistFunc::rPoisson
+"Realization of a Poisson distribution.
+
+Parameters
+----------
+lambda: float, :math:`\lambda\geq 0`
+    The intensity of the Poisson distribution
+size : int
+    The number of realizations to generate.
+
+Returns
+-------
+realizations : int or :class:`~openturns.Indices`
+    The realizations of the discrete disctribution.
+
+Notes
+-----
+For the small values of :math:`\lambda`, we use the method of inversion by
+sequential search described in [devroy1986]_ and with the important errata in
+[devroy1986b]_. For the large values of :math:`\lambda`, we use the ratio of
```

Comment:
devroye1986b

Reply:
Fixed. Thanks, this error drove me crazy!